libs/version/README.md

```diff
@@ -0,0 +1,13 @@
+# Libs/Version
+
+This library consists of a single function "getVersion" which returns the version of the application that called it.
```

Comment:
Nit: does it worth pointing out that this lib consists of a single function? (In this case if we add a second utility function then we need to update the README - which is not the end of the world ofc.)